app.rb

```diff
@@ -1,8 +1,23 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/player'
+
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+    
+  get '/' do    
+    erb :index
   end
 
+  post '/names' do
+    $player_1 = Player.new(params[:player_1_name])
+    redirect '/play'
+  end 
+
+  get '/play' do
+    @name1 = $player_1.name
+    erb :play
```

Comment:
Referring to play view, but erb file is play_game.erb which you'd already flagged.